src/test/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperationsTest.scala

```diff
@@ -2361,8 +2362,8 @@ class TezosDatabaseOperationsTest
 
         val result = dbHandler.run(populateAndTest.transactionally).futureValue
         result shouldBe List(
```

Comment:
that does not look right, we are ordering by hash descending and after change of postgres version it works different way than expected

Reply:
It depends on expectations for String sorting.
Do we expect all upper-case chars to come first or do we mix upper/lower?

My understanding is that it all depends on the encoding done by postgres for comparison of chars, which is decided usually on the database setup.

I simply experienced that the test would work on a previous version of postgres container and would break on the 11.x series, giving inverted ordering.
Therefore I tried to figure out how those differ wrt sorting rows, but I only found that the sorting depends on COLLATION of the column

>Character-string data is sorted according to the collation that applies to the column being sorted. That can be overridden at need by including a COLLATE clause in the expression, for example ORDER BY mycolumn COLLATE "en_US". For more information see Section 4.2.10 and Section 23.2.

If I get it correctly we use UTF-8 compatible sorting based on characters' code points.

Whatever the solution, we must ensure that the database ORDER BY works correspondingly and consistently on every deployment/environment.

See: https://www.postgresql.org/docs/11/collation.html (§23.2.2)
https://www.postgresql.org/docs/11/sql-expressions.html#SQL-SYNTAX-COLLATE-EXPRS

Reply:
If I remember correctly I was fixing this inconsistency issue some time ago(it either worked on travis or locally) - that's why in previous version of `InMemoryDatabase` we had something like this:
```
  /* turns off anti-corruption guarantees settings that will improve performance on testing
   * override to change or add test-specific settings
   */
  protected val pgInitParams = List("--nosync", "--lc-collate=C")
```

Reply:
The `nosync` option is already set by default by testcontainers.

I will see how to restore the collation option too, to make the behaviour like before

Reply:
Fixed by avoiding the 'alpine' based image.
Configuration parameters might have changed names between versions, e.g. `nosync`